src/main/java/org/broadinstitute/hellbender/engine/TwoPassVariantWalker.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.hellbender.engine;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.engine.filters.CountingReadFilter;
+import org.broadinstitute.hellbender.engine.filters.VariantFilter;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.stream.StreamSupport;
+
+public abstract class TwoPassVariantWalker extends VariantWalker {
+
+    @Override
+    public void traverse(){
+        final VariantFilter variantFilter = makeVariantFilter();
```

Comment:
Can you file a github ticket saying that `VariantWalkerBase` (and subclasses) need to migrate to `CountingVariantFilter`, so that summary statistics for variant filtration can be printed at the end of traversal, as they are for read filters currently? 

@cmnbroad Do you know why `CountingVariantFilter` never got hooked up here?

Reply:
I'm not sure why we didn't do that.

Reply:
done